src/components/RangeInput.vue

```diff
@@ -1,15 +1,20 @@
 <template>
 <div v-if="state" :class="suit()">
-  <form :class="suit('form')" @submit.prevent="refine()">
+  <slot
+    :currentRefinements="this.state.start"
+    :refine="this.refine"
+  >
+  <form :class="suit('form')" @submit.prevent="refine(minInput, maxInput)">
     <label :class="suit('label')">
-      <input :class="[suit('input'), suit('input', 'min')]" type="number" ref="minInput" :min="this.min" :placeholder="this.min"/>
+      <input :class="[suit('input'), suit('input', 'min')]" type="number" :min="this.min"  :placeholder="this.min" :value="this.state.start && this.state.start[0]" @change="minInput = $event.currentTarget.value"/>
```

Comment:
The `min` & `max` value should be the one of the range rather than the provided values. We can also use this range to set the placeholder (see [React](https://community.algolia.com/react-instantsearch/storybook/?selectedKind=RangeInput&selectedStory=default&full=0&addons=1&stories=1&panelRight=1&addonPanel=storybooks%2Fstorybook-addon-knobs) & [IS](https://community.algolia.com/instantsearch.js/v2/dev-novel/?selectedStory=RangeInput.default) behaviour).